docs/conf/help.example.conf

```diff
@@ -1109,55 +1109,50 @@ setting +I <extban>.
 
 Matching extbans:
 
- a:<mask>      Matches user with both a matching banmask and real name,
-               where <mask> is in the format nick!user@host+realname
-               (requires the realnameban module).
- j:<channel>   Matches anyone in the given channel. Does not support
-               wildcards (requires the channelban module).
- n:<class>     Matches users in a matching connect class (requires
-               the classban module).
- o:<pattern>   Checks whether users are logged into a server operator
-               account with a name matching <pattern> (requires the
-               operchans module).
- r:<realname>  Matches users with a matching real name (requires the
-               realnameban module).
- s:<server>    Matches users on a matching server (requires the
-               serverban module).
- z:<tlsfp>     Matches users having the given TLS certificate
-               fingerprint (requires the sslmodes module).
- w:<gateway>   Matches users who are connecting via a WebIRC gateway that
-               matches <gateway>.
- G:<pattern>   Matches against the two letter country code for the
-               country that users are connecting from (requires the
-               geoban module).
- O:<opertype>  Matches server operators of a matching type, mostly
-               useful as an invite exception (requires the
-               operchans module).
- R:<account>   Matches users logged into a matching account (requires
-               the account module).
- U:<banmask>   Matches unregistered users matching the given banmask.
-               (requires the account module).
+ realmask:<mask>, a:<mask>          Matches user with both a matching banmask and real name, where
+                                    <mask> is in the format nick!user@host+realname (requires the
+                                    realnameban module).
+ channel:<channel>, j:<channel>     Matches anyone in the given channel. Does not support wildcards
+                                    (requires the channelban module).
```

Comment:
This does already support wildcards (not a change you made but its worth fixing whilst touching this).

Reply:
Fixed!